### ISSUE TITLE
Calling 110 on China Telecom fails and causes the phone to lose reception until it is rebooted

### ISSUE THREAD

**Issue by fffelix-jan** [kyujin-cho/pixel-volte-patch#327]
**Describe the bug**
Calling 110 (the police emergency number in China) on China Telecom fails and causes the phone to lose reception until it is rebooted. (This might also apply to the 119 fire and 120 ambulance numbers, but luckily, I've never had to call them so far.) This bug may be dangerous in a life-threatening condition.

**To Reproduce**
1. Be in China, use a China Telecom SIM card and enable VoLTE using Pixel IMS.
2. Open the Phone app.
3. Call 110.
4. The phone fails to call 110 and displays "No Service" until the phone is rebooted.

**Expected behavior**
The police are called. (This issue can be worked around by calling `<areaCode> 110`, which I can confirm works in Hangzhou by dialing 0571 110.)

**Smartphone (please complete the following information):**
 - Device: Pixel 6
 - OS: GrapheneOS Android 15 (latest version)
 - LTE-only mode (2G and 3G are disabled)

**Additional context**
I have a feeling it's due to the special way the phone handles emergency numbers.

**Comment by fffelix-jan** [comment]
Reminder: 110 is an EMERGENCY NUMBER! Please do not call them if there is no emergency!

**Comment by UjuiUjuMandan** [comment]
Yeah, facing the same problem. I have to add area code before the emergency number.

And for testing, kindly call 122 (traffic accident report) Instead of 110.

**Comment by Blacky-cn** [comment]
> Yeah, facing the same problem. I have to add area code before the emergency number.
> 
> And for testing, kindly call 122 (traffic accident report) Instead of 110.

I learned about this issue when I wanted to buy Pixel. After adding the area code before the emergency number, can the call go through normally?

**Comment by UjuiUjuMandan** [comment]
> can the call go through normally?

Read the post again carefully before wasting my inbox and time any further. Thanks.

**Comment by Blacky-cn** [comment]
> > can the call go through normally?
> 
> Read the post again carefully before wasting my inbox and time any further. Thanks.

You said you “have to add area code before the emergency number”, so I want to know if this method really works, and if it's still effective now.
Please think carefully about my question before replying to me, don't waste my time.
